Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.9"
+version = "1.9.1"
```

Minor change.
Those is a feature not a bug fix


```suggestion
version = "1.10.0"
```